app/code/Magento/NewsletterGraphQl/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "magento/module-newsletter-graph-ql",
+    "description": "N/A",
+    "config": {
+        "sort-packages": true
+    },
+    "require": {
+        "php": "~7.1.3||~7.2.0",
+        "magento/framework": "*",
+        "magento/module-backend": "*",
+        "magento/module-cms": "*",
+        "magento/module-customer": "*",
+        "magento/module-eav": "*",
+        "magento/module-email": "*",
+        "magento/module-require-js": "*",
```

Comment:
why is there a hard dependency on requireJS module?

Reply:
Removed dependency on requireJS